src/UI/examples/Table/Presentation/base.php

```diff
@@ -0,0 +1,64 @@
+<?php
+function base() {
+	global $DIC;
+	$f = $DIC->ui()->factory();
+	$renderer = $DIC->ui()->renderer();
+
+	//build viewcontrols
+	$actions = array ("All" => "#",	"Upcoming events" => "#");
+	$aria_label = "filter entries";
+	$view_controls = array(
+		$f->viewControl()->mode($actions, $aria_label)->withActive("All")
+	);
+
+	//build table
+	$ptable = $f->table()->presentation(
+		'Presentation Table', //title
+		$view_controls,
+		function ($row, $record, $ui_factory, $environment) { //mapping-closure
```

Comment:
This mapping closure definitely need more explanation in the interface and/or resp. factory (you may also just reference from one to the other, to not generate redundant explanations). Shortly note the params, and what they do. You probably should also reference this example here. You should also explain the params this closure gets from the internas. E.g. How could record be used, where is it passed. etc. Obviously the example is extremely telling. But I believe some abstract description also to be necessary.

Reply:
Tried to. Better now?